app/controllers/articles_controller.rb

```diff
@@ -26,6 +26,10 @@ def feed
                   handle_user_or_organization_feed
                 elsif params[:tag]
                   handle_tag_feed
+                elsif request.path == latest_feed_path
+                  @articles.where(featured: true)
```

Comment:
Sorry for not being clearer before but `where(featured: true)` is the condition we're trying to remove, because we want the latest articles, not the feature ones. As I mentioned in the other discussion, we need to replicate the same query we use for `dev.to/latest`, so 

https://github.com/forem/forem/blob/90aa6932040f584467c971456b56500e4d333c72/app/controllers/articles_controller.rb#L24

plus the score filter. That's it. Featured is used in the `else` clause, which is the one of the general feed, not the "latest" feed.

Currently we have these feeds:

- "username feed" with the list of all published articles created by the user (`Article.feed` + `handle_user_or_organization_feed`)
- "organization feed" with the list of all published articles created by the organization (`Article.feed` + `handle_user_or_organization_feed`)
- "tag feed" with the list of all published articles with the given tag (`Article.feed` + `handle_tag_feed`)
- "generic feed" (the `else` clause) with the list of published articles that are featured (`Article.feed` + `where(featured: true)`)

What we want to add is "the list of all published articles above a certain score", so (`Article.feed` + the score filter).

We do not want to filter by `featured` here, otherwise it'd already be the current "generic feed"

Let me know if anything is unclear

Reply:
Oh okay, thanks a lot for the detailed explanation! :smiley: But if I am understanding right, at https://github.com/forem/forem/pull/11956#discussion_r545432001 @juliannatetreault suggested me otherwise on the presence of 'not featured articles' on `feed/latest`

Reply:
Consider that right now the only difference between the two paths is the presence or absence of articles with a lower score, which is not what https://dev.to/latest is

In https://github.com/forem/forem/issues/8940 @todsac mentioned that:

> The only way to ensure you’re getting the latest articles right now is to increment the API

that's because article ids are incremental, so to get the latest you have to come up with a trick to fetch the next article by ID

I might have misunderstood the initial goal, and I think we're in this empasse because the issue itself isn't 100% clear and devoid of a discussion but I don't see the added value to the OP to add a new route which essentially returns the same data.

cc @juliannatetreault for her feedback

Thank you for the patience!